parameter/ParameterBlackboard.h

```diff
@@ -52,6 +52,34 @@ class CParameterBlackboard : private utility::NonCopyable
     void writeBuffer(const void* pvSrcData, size_t size, size_t offset);
     void readBuffer(void* pvDstData, size_t size, size_t offset) const;
 
+    /**
+     * Raw write the blackboard memory.
+     * May be used to write a configurable element's settings
+     *
+     * @param[in] bytes the source data bytes vector.
+     * @param[in] offset the destination offset in the blackboard.
+     *
+     * Notes:
+     *    - Endianness is not handled by the method and thus data are written in the blackboard
+     *      with same representation as the one passed.
+     *    - This function will assert if the bytes size + offset exceeds the size of the blackboard iself.
```

Comment:
This function *asserts* that the bytes size + offset exceeds the size of the blackboard iself.